docs/source/faq.rst

```diff
@@ -0,0 +1,113 @@
+across multiple server nodes in cluster deployment. To resolve this issue you can turn off level2 cache by setting
+``datanucleus.cache.level2.type`` property to ``none``. There may be slight performance drop per server in result of
+turning off level2 cache. Level1 cache should be kept as it works within one transaction, therefore not affecting
+cluster deployment.
+
+How to use SSL for database connections in MOTECH?
+--------------------------------------------------
+In order to establish secure SSL connection JDBC URL needs to be updated with JDBC driver property ``'useSSL'=true'``
+eg. if you are using mySQL at localhost JDBC URL should be ``jdbc:mysql://localhost:3306/?useSSL=true``. As the URL
+entered on the MOTECH bootstrap page is only the base for actual database connections urls, you should provide URL with
+SSL parameter to appropriate config files.
+
+For MDS it can be set by editing ``~/.motech/config/datanucleus.properties`` and changing the property ``javax.jdo.option.ConnectionURL``.
+For quartz you have to edit the variable ``org.quartz.dataSource.motechDS.URL`` in ``quartz.properties``. If your config
+source is FILE, then you have to edit the file in scheduler directory in ``~/.motech/config``. If your source is UI, you
+have to change the quartz configuration through the manage modules section in the Admin UI.
+
+MOTECH is not working as expected, how can I check what's wrong?
+----------------------------------------------------------------
+All the problems that MOTECH encounters are logged. The logs can be checked in two ways. If MOTECH is working you can
+access logs by choosing ``Server Log`` from ``Admin`` panel on the UI. There you can also specify logging level for
+specific packages - this is done by selecting wanted log level in ``Server Log -> Log options`` panel.
+
+If access to UI is not possible, for standard MOTECH installation on Tomcat logs can be also found in ``catalina.out``
+file located in Tomcat installation logs folder. For example for MOTECH deployed on Tomcat 7.0.62 the ``catalina.out``
+is located in ``~/apache-tomcat-7.0.62/logs`` folder. The content of the log file can be displayed by any text editor
+or in Ubuntu console by for example ``less catalina.out`` command.
+
+I am getting blueprint timeout error - what does it mean?
+---------------------------------------------------------
+Blueprint timeout error is encountered when one of the modules has reference to the service from another module defined in
+``blueprint.xml`` file and that service has not been made available. By default module waits 5 minutes for required service,
+if in this time service won't be delivered the timeout error will be thrown. First step in resolving blueprint timeout
+errors should be checking if the problematic service is correctly registered as OSGi service. Below you can see an example
+of properly defined OSGi service exporting:
```

Comment:
remove "exporting", it doesn't make sense here, just end the sentence on service